lib/Service/VerdictService.php

```diff
@@ -141,6 +141,7 @@ private function tagFile(int $fileId, string $tagName) {
 	 */
     public static function isFileTooLargeToScan(string $path): bool {
         $size = filesize($path);
+        if ($size === 0) { return false; }
         return !$size || $size > self::MAX_FILE_SIZE;
```

Comment:
The problem would be still present for a null-value in the size.

```php
<?php

$foobar = null;
echo var_export(!$foobar); // true

$foobar = 0;
echo var_export(!$foobar); // true
```

Reply:
The functions has PHPDocs that tell us it can only return an integer or "false" on any failure.

Reply:
Yeah. You are right. Still would prefer my suggestions. Error case above the return.